codegen/src/main/java/software/amazon/awssdk/codegen/poet/client/DelegatingAsyncClientClass.java

```diff
@@ -25,24 +25,27 @@
 import com.squareup.javapoet.MethodSpec;
 import com.squareup.javapoet.TypeSpec;
 import javax.lang.model.element.Modifier;
-import software.amazon.awssdk.annotations.SdkInternalApi;
+import software.amazon.awssdk.annotations.SdkPublicApi;
 import software.amazon.awssdk.codegen.model.config.customization.UtilitiesMethod;
 import software.amazon.awssdk.codegen.model.intermediate.IntermediateModel;
 import software.amazon.awssdk.codegen.model.intermediate.OperationModel;
 import software.amazon.awssdk.codegen.poet.PoetExtension;
 import software.amazon.awssdk.codegen.poet.PoetUtils;
 import software.amazon.awssdk.codegen.utils.PaginatorUtils;
+import software.amazon.awssdk.core.SdkClient;
+import software.amazon.awssdk.utils.Validate;
 
 public class DelegatingAsyncClientClass extends AsyncClientInterface {
 
+    private static final String DELEGATE = "delegate";
```

Comment:
This feels kind of silly. Should we just inline it? I think the principal of not having magic constants applies to where the constants aren't self-describing. E.g. 24 instead of HOURS_PER_DAY. This particular case feels redundant.

Reply:
reverted back to inline